ts/features/bonus/cgn/analytics/index.ts

```diff
@@ -0,0 +1,44 @@
+import { getType } from "typesafe-actions";
+import { cgnEnabled } from "../../../../config";
+import { mixpanel } from "../../../../mixpanel";
+import { Action } from "../../../../store/actions/types";
+import { getNetworkErrorMessage } from "../../../../utils/errors";
+import {
+  cgnActivationComplete,
+  cgnActivationStart,
+  cgnActivationCancel,
+  cgnActivationBack,
+  cgnActivationStatus,
+  cgnRequestActivation
+} from "../store/actions/activation";
+import { cgnDetails } from "../store/actions/details";
+
+const trackAction = (mp: NonNullable<typeof mixpanel>) => (
+  action: Action
+): Promise<any> => {
+  switch (action.type) {
+    case getType(cgnActivationStart):
+    case getType(cgnActivationComplete):
+    case getType(cgnActivationCancel):
+    case getType(cgnActivationBack):
+    case getType(cgnActivationStatus.request):
+    case getType(cgnActivationStatus.success):
+    case getType(cgnRequestActivation.request):
+    case getType(cgnRequestActivation.success):
```

Comment:
```suggestion
    case getType(cgnRequestActivation):
```

Since this action has been changed in fbceb75 now it is a Standard action and not an Async one

Same comment is valid for the `cgnRequestActivation.request` action it can be removed